## Description: Don't maximize the window at startup
## Origin/Author: Lenik (谢继雷) <xjl@99jsj.com>
Index: amule-2.2.6+debian0/src/amuleDlg.cpp
===================================================================
--- amule-2.2.6+debian0.orig/src/amuleDlg.cpp	2010-12-30 21:42:53.647720000 +0800
+++ amule-2.2.6+debian0/src/amuleDlg.cpp	2010-12-30 21:42:34.191719999 +0800
@@ -914,7 +914,7 @@
	int x2 = config->Read(section + wxT("MAIN_X_SIZE"), -1);
	int y2 = config->Read(section + wxT("MAIN_Y_SIZE"), -1);

-	int maximized = config->Read(section + wxT("Maximized"), 01);
+	int maximized = config->Read(section + wxT("Maximized"), 0l);

	// Kry - Random usable pos for m_srv_split_pos
	m_srv_split_pos = config->Read(section + wxT("SRV_SPLITTER_POS"), 463l);
